recipes-apps/xcsoar/xcsoar_7.21.bb

```diff
@@ -42,14 +42,13 @@ S = "${WORKDIR}/git"
 
 LC_LOCALE_PATH = "/usr/share/locale"
 
-SRC_URI = " \
-    git://github.com/XCSoar/XCSoar.git;protocol=git;tag=v${PV} \
+SRC_URI = "git://github.com/XCSoar/XCSoar.git;protocol=git;tag=v${PV} \
     file://ov-xcsoar.conf "
     
-# SRC_URI_append = "file://0001_no-version-lua.patch"
-SRC_URI_append = "file://0001-avoid-tail-cut.patch"
-SRC_URI_append = "file://0005-Adapted-toolchain-prefixes-for-cross-compile.patch"
-SRC_URI_append = "file://0007-Disable-touch-screen-auto-detection.patch"
+# with SRC_URI_append you can easy switch between the various patches!
```

Comment:
What do you mean with "switch between patches" ??

Reply:
This is only a help for the editor/developer:
If you write all files (and patches) in the SRC_URI setting you cannot comment out a patch! You have to move the line outside from the setting, delete the line appender '\' - and then you can comment out 
With this SRC_URI_append syntax the patch is switched on and of very easy with the comment character.

Reply:
ok